### DISCUSSION TITLE
Thermostat code for Nano RP2040 Connect

### DISCUSSION THREAD

**Discussion by pwl1821** [micropython/micropython#9499] [Using MicroPython]
Hi,
Can anyone suggest code for thermostatic control using an Arduino Nano RP2040 Connect?

The initial application is On/Off cooling control to drive 3.3v relays of ANC GPIO's. Code for an adjustable deadband of 0.6C or greater would be helpful.  

UI at this point can be via Thonny or equivalent. Thoughts and/or code for UI on a phone or laptop would be appreciated.

No modulating / PID control.  Heating or heating / cooling control also welcome. 

Any thoughts on starting points?

**Comment by 2dof** [comment]
For on-of control you shout try  realy (relay with hysteresis    ) contol , it can be done without pid (pi) control. 
 See some  example:
https://m.youtube.com/watch?v=GW4YsLnrO2s
